src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -69,8 +69,8 @@ export const ArtworkApp: React.SFC<Props> = props => {
           </Tabs>
         </Col>
       </Row>
-      <Row mb={6}>
-        <Col xl={8} lg={8} md={8} sm={12} xs={12}>
+      <Row>
+        <Col xl={8} lg={8} md={8} sm={8} xs={12}>
```

Comment:
@oxaudo - Idiomatically, breakpoints like this should go from smallest to largest (left to right) and note that by default it's 12, and a breakpoint set differently will expand up. This is hard to explain in words, but by example it should make sense and can be simplified to just one prop: 

`<Col sm={8} />` 

Does this make sense?

Reply:
Cool. Did not realize that. This code was there before, but what you are saying makes total sense. Will update with next change.

Reply:
👍